src/matter/session/secure/SecureChannelMessenger.ts

```diff
@@ -60,13 +83,23 @@ export class SecureChannelMessenger<ContextT> {
         await this.exchange.send(MessageType.StatusReport, writer.toByteArray());
     }
 
-    protected throwIfError(messageType: number, payload: ByteArray) {
-        if (messageType !== MessageType.StatusReport) return;
+    decodeStatusReport(payload: ByteArray) {
```

Comment:
Now that Schema exists, you can replace decodeStatusReport and part of sendStatusReport with a StatusReportSchema.encode/decode

Reply:
No we do not have the SecureChannel StatusResponse as Tlv Schema till now